dashboard/app/mailers/pd/teacher_application_mailer.rb

```diff
@@ -15,6 +15,8 @@ def principal_approval_request(teacher_application)
     @teacher_name = teacher_application.teacher_name
     @program_name = teacher_application.program_name
     @program_url = teacher_application.program_url
+    @due_date = @program_name == Pd::Workshop::COURSE_CSP ? 'May 5, 2017' : 'April 17, 2017'
```

Comment:
Do we send this for other courses? (Admin, CS in A, etc). if not, make this an else if and throw some error if the course is unrecognized somewhere